### PR TITLE
test(Wallet) Manage seed phrase account(imported and generated) and k…

### PR DESCRIPTION
**Pull request by Valentina1133**
https://ethstatus.testrail.net/index.php?/cases/view/703030
https://ethstatus.testrail.net/index.php?/cases/view/703036
https://ethstatus.testrail.net/index.php?/cases/view/703514
Screenshot 2023-09-07 at 11 35 19

### CODE REVIEW

File: test/ui-pytest/gui/screens/settings.py [hunk 1]
```diff
@@ -75,3 +76,37 @@ def get_community_info(self, name: str) -> UserCommunityInfo:
...
+            'settingsContentBaseScrollView_setupFromExistingKeycardAccount_StatusListItem')
+        self._create_new_keycard_account_button = Button(
+            'settingsContentBaseScrollView_createNewKeycardAccount_StatusListItem')
+        self._import_restore_via_seed_phrase_button = Button(
+            'settingsContentBaseScrollView_importRestoreKeycard_StatusListItem')
+        self._import_from_keycard_button = Button('settingsContentBaseScrollView_importFromKeycard_StatusListItem')
+        self._check_whats_on_keycard_button = Button(
+            'settingsContentBaseScrollView_checkWhatsNewKeycard_StatusListItem')
+        self._factory_reset_keycard_button = Button('settingsContentBaseScrollView_factoryResetKeycard_StatusListItem')
+
```
Comment (StateOf-Vlado) [inline test/ui-pytest/gui/screens/settings.py:96]: Please make symbolic names shorter and maybe names of variables.
It is good to fir one class attribute in one line.

File: test/ui-pytest/tests/test_wallet/__init__.py
Comment (StateOf-Vlado) [inline test/ui-pytest/tests/test_wallet/__init__.py:1]: we don't need init file in test folder, please remove it

File: test/ui-pytest/gui/screens/settings.py (hunk 1 above)
Comment (Valentina1133) [reply test/ui-pytest/gui/screens/settings.py:92]: Done

File: test/ui-pytest/tests/test_wallet/__init__.py
Comment (Valentina1133) [reply test/ui-pytest/tests/test_wallet/__init__.py:1]: Removed it

File: test/ui-pytest/tests/test_wallet/test_wallet.py
[the remark this answers is not in the document]
Comment (Valentina1133) [reply test/ui-pytest/tests/test_wallet/test_wallet.py:1]: There is only one test for now that's why I left general name. But ok, I renamed it to saved addresses, will rename it later if there will be more tests

**Comment by Valentina1133** [comment]
Fixed everything, created a new PR in new repo https://github.com/status-im/desktop-qa-automation/pull/103/
Closing this one

**Pull request closed without merging**